src/vivarium_cluster_tools/psimulate/results/cli_options.py

```diff
@@ -6,12 +6,32 @@
 Command line options for configuring results handling in psimulate runs.
 
 """
+
 import click
 
 from vivarium_cluster_tools.cli_tools import MINUTES_OR_NONE
 
+DEFAULT_BATCH_SIZE = 200
+DEFAULT_CHUNK_SIZE = 500 * 1024 * 1024  # 500 MB in bytes
+
+with_batch_size = click.option(
+    "--batch-size",
+    type=int,
+    default=DEFAULT_BATCH_SIZE,
+    show_default=True,
+    help="Number of simulation jobs to accumulate before writing results to disk.",
+)
+with_chunk_size = click.option(
+    "--chunk-size",
```

Comment:
Should we call this something `"--output-file-size"`? It seems like we could be more explicit that this is for file sizes. 

I think people will get confused about "chunks" and "batches"